components/list/test/list-item-drag-handle.visual-diff.js

```diff
@@ -42,7 +43,6 @@ describe('d2l-list-item-drag-handle', () => {
 		});
 
 		itWithReload('keyboard-mode', async function() {
-			await focusMethod('d2l-list-item-drag-handle');
```

Comment:
This wasn't working but was also redundant with the earlier tests for focus, so I just removed focusing on the drag handle.